src/gateways/accountHolderApiGateway.ts

```diff
@@ -28,4 +30,26 @@ export class AccountHolderApiGateway {
       throw error;
     }
   }
+
+  public async getAccountHolderDetails(
+    accountHolderId: string,
+    accessToken: string
+  ): Promise<IAccountHolderDetailsResponseBody> {
+    const url = `${this.apiUrl}/${this.accountHolderDetailsEndpoint}/${accountHolderId}`;
+    try {
+      const response = await axios.get<
+        any,
+        AxiosResponse<IAccountHolderDetailsResponseBody>
+      >(url, {
+        headers: { Authorization: `Bearer ${accessToken}` },
+      });
+
+      console.log("response", response);
```

Comment:
Not sure if this console.log is here intentionally so thought I'd point it out!

Reply:
removed